src/spirc.rs

```diff
@@ -246,7 +295,9 @@ impl SpircTask {
 
         if frame.get_ident() == self.ident ||
            (frame.get_recipient().len() > 0 && !frame.get_recipient().contains(&self.ident)) {
-            return;
+            if frame.get_typ() == MessageType::kMessageTypeNotify {
```

Comment:
@plietar I've tested with the following code snippet:

``` Rust
fn handle_command(&mut self, cmd: SpircCommand) {
    SpircCommand::Pause => {
        if self.device.get_is_active() {
            self.handle_pause();
        } else {
             CommandSender::new(self, MessageType::kMessageTypePause).send();
        }
    }
}
```

librespot stops playback, but the Spotify Player is not notified about it. The Play/Pause button is still showing the pause icon and the progress bar for the seconds is still increasing.

So, these messages needs to be broadcasted so the Spotify Player is notified about the changes, right?